File: app/src/main/java/sk/romanstrazanec/arkanoid/GameObjects/Brick.java
package sk.romanstrazanec.arkanoid.GameObjects;

import android.graphics.Canvas;
import android.graphics.Paint;

public class Brick extends GameObject {
    private float width, height;

    public Brick(float x, float y, float width, float height, int color) {
        super(x, y, color);
        this.width = width;
        this.height = height;
    }

    public float x2(){
        return x + width;
    }

    public float y2(){ return y + height; }

    @Override
    public void draw(Canvas canvas, Paint paint){
        super.draw(canvas, paint);
        canvas.drawRect(x, y, x + width, y + height, paint);
    }
}

File: app/src/main/java/sk/romanstrazanec/arkanoid/gameobjects/Brick.kt
package sk.romanstrazanec.arkanoid.gameobjects

import android.graphics.Canvas
import android.graphics.Paint

class Brick(x: Float, y: Float, private val width: Float, private val height: Float, color: Int) : GameObject(x, y, color) {
    fun x2(): Float {
        return x + width
    }

    fun y2(): Float {
        return y + height
    }

    override fun draw(canvas: Canvas?, paint: Paint) {
        super.draw(canvas, paint)
        canvas!!.drawRect(x, y, x + width, y + height, paint)
    }
}